### PR TITLE
[ADD][stock_kardex]-history of movements of products

### PR DESCRIPTION
**Pull request by SaraiOsorio**
@alan196

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please add a proper README according OCA guidelines: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

**Comment by rousseldenis** [comment]
@SaraiOsorio What's the status of this ?

File: stock_kardex/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        'views/report_stock.xml',
+        'views/search_template_view.xml',
+        'views/res_config_settings_view.xml',
+    ],
+    'qweb': [
+        'static/src/xml/stock_kardex_template.xml',
+    ],
+    'auto_install': True,
+    'installable': True,
+    'license': 'OEEL-1',
```
Comment (rousseldenis) [inline stock_kardex/__manifest__.py:26]: @SaraiOsorio Is it the right licensing ?

**Comment by rousseldenis** [comment]
@SaraiOsorio Do you plan to finish this ?

**Pull request closed without merging**